src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -572,6 +573,17 @@ public function testUnionTypeDeserializable()
         $this->assertEquals(new DummyUnionType(), $actual, 'Union type denormalization third case failed.');
     }
 
+    /**
+     * @requires PHP 8.2
+     */
+    public function testFalseBuiltInTypes()
```

Comment:
This test case was triggering a warning: `This test did not perform any assertions`

I fixed it.